release/src/main/scripts/build_release_candidate.sh

```diff
@@ -0,0 +1,220 @@
+cd ~
+mkdir ${LOCAL_JAVA_STAGING_DIR}
+cd ${LOCAL_JAVA_STAGING_DIR}
+svn co ${ROOT_SVN_URL}
+mkdir beam/${RELEASE}
+cd beam/${RELEASE}
+
+echo "----------------Downloading Source Release-------------------"
+echo "Downloading: ${GIT_BEAM_ARCHIVE}/release-${RELEASE}.zip"
+wget ${GIT_BEAM_ARCHIVE}/release-${RELEASE}.zip  -O apache-beam-${RELEASE}-source-release.zip
+
+echo "----Signing Source Release apache-beam-${RELEASE}-source-release.zip-----"
+gpg --armor --detach-sig apache-beam-${RELEASE}-source-release.zip
+
+echo "----Creating Hash Value for  apache-beam-${RELEASE}-source-release.zip----"
+sha512sum apache-beam-${RELEASE}-source-release.zip > apache-beam-${RELEASE}-source-release.zip.sha512
+
+svn add beam/${RELEASE}
+svn status
+echo "Please confirm these changes are ready to commit: [y|N] "
+read confirmation
+if [[ $confirmation != "y" ]]; then
+  echo "Exit without staging source release on dist.apache.org."
+  rm -rf ~/${LOCAL_JAVA_STAGING_DIR}
+  exit
+fi
+svn commit
+rm -rf ~/${LOCAL_JAVA_STAGING_DIR}
+
+echo "============Staging Python Binaries on dist.apache.org========="
+cd ~
+mkdir ${LOCAL_PYTHON_STAGING_DIR}
+cd ${LOCAL_PYTHON_STAGING_DIR}
+
+echo '-------------------Cloning Beam Release Branch-----------------'
+git clone ${GIT_REPO_URL}
+cd ${BEAM_ROOT_DIR}
+git checkout ${RELEASE_BRANCH}
+
+echo '-------------------Generating Python Artifacts-----------------'
+cd sdks/python
+python setup.py sdist --format=zip
+cd dist
+
+svn co https://dist.apache.org/repos/dist/dev/beam
+mkdir beam/${RELEASE}/${PYTHON_ARTIFACTS_DIR}
+cp apache-beam-${RELEASE}.zip beam/${RELEASE}/python_test/apache-beam-${RELEASE}.zip
+cd beam/${RELEASE}/${PYTHON_ARTIFACTS_DIR}
+
+echo "------Signing Source Release apache-beam-${RELEASE}.zip------"
+gpg --armor --detach-sig apache-beam-${RELEASE}.zip
+
+echo "------Creating Hash Value for apache-beam-${RELEASE}.zip------"
+sha512sum apache-beam-${RELEASE}.zip > apache-beam-${RELEASE}.zip.sha512
+
+cd ..
+svn add ${PYTHON_ARTIFACTS_DIR}
+svn status
+echo "Please confirm these changes are ready to commit: [y|N] "
+read confirmation
+if [[ $confirmation != "y" ]]; then
+  echo "Exit without staging python artifacts on dist.apache.org."
+  rm -rf ~/${PYTHON_ARTIFACTS_DIR}
+  exit
+fi
+svn commit
+rm -rf ~/${PYTHON_ARTIFACTS_DIR}
+
+echo "==============Creating PR for Updating Website==============="
+cd ~
+mkdir ${LOCAL_WEBSITE_UPDATE_DIR}
+cd ${LOCAL_WEBSITE_UPDATE_DIR}
+mkdir ${LOCAL_PYTHON_DOC}
+mkdir ${LOCAL_JAVA_DOC}
+mkdir ${LOCAL_WEBSITE_REPO}
+
+echo "------------------Building Python Doc------------------------"
+cd ${LOCAL_PYTHON_DOC}
+pip install tox
+git clone ${GIT_REPO_URL}
+cd ${BEAM_ROOT_DIR}
+git checkout ${RELEASE_BRANCH}
+cd sdks/python && tox -e docs
+GENERATED_PYDOC=~/${LOCAL_WEBSITE_UPDATE_DIR}/${LOCAL_PYTHON_DOC}/${BEAM_ROOT_DIR}/sdks/python/target/docs/_build
+
+echo "----------------------Building Java Doc----------------------"
```

Comment:
This sections looks different compared to: https://beam.apache.org/contribute/release-guide/#build-a-release-candidate

(The part `...To update the website, you must unpack this jar file from the release candidate into the source tree of the website....`)

Reply:
I'm not sure which URL we can use to get jar files. So there are 2 possible ways for this section:
1. Find the right URL.
2. We can clone the source code from release branch and run gradle to generate javadoc.
WDYT?

Reply:
I would guess that running it locally would result in building the same jar file `org.apache.beam:beam-sdks-java-javadoc` to be generated. (This artifact already needs to be one of the staged artifacts as part of the release.

Reply:
Fixed in commit https://github.com/apache/beam/pull/6059/commits/e1d1f09ce85186808014bbc47b46127c01679f78